addons/web/static/src/views/fields/properties/property_definition.xml

```diff
@@ -149,8 +149,8 @@
                     </td>
                 </tr>
                 <tr t-if="!['tags', 'selection'].includes(state.propertyDefinition.type)
-                    &amp;&amp; (!['many2one', 'many2many'].includes(state.propertyDefinition.type)
-                    || (state.propertyDefinition.comodel &amp;&amp; state.propertyDefinition.comodel.length))"
+                    and (!['many2one', 'many2many'].includes(state.propertyDefinition.type)
+                    || (state.propertyDefinition.comodel and state.propertyDefinition.comodel.length))"
```

Comment:
If you replace those with "and" then we should replace the "||" by "or" as well, more consistent, right?

Reply:
I'm not sure, I try to see some example in /web, and looks like they use "and" but "||"
- odoo/addons/web/static/src/views/fields/reference/reference_field.xml
- odoo/addons/web/static/src/views/fields/selection/selection_field.xml
- odoo/addons/web/static/src/views/fields/url/url_field.xml
(3 first random template I openned follow that rules)

Reply:
That is super strange, but ok :smile:

Reply:
yes I don't like that to ^^ (especially the mix of js / python which is super confusing)